src/Generators/OrmTagGenerator.php

```diff
@@ -155,7 +157,11 @@ protected function generateTagsForDataLists($fields, $listType = DataList::class
                 $listName = $this->getAnnotationClassName($listType);
                 $dataObjectName = $this->getAnnotationClassName($dataObjectName);
 
-                $tagString = "{$listName}|{$dataObjectName}[] {$fieldName}()";
+                if ($useGenerics) {
```

Comment:
I would remove the else bit, and leave line 158 intact. That way, it's set, then overridden if the else is true.
Just a personal preference of avoiding elses where possible

Reply:
agreed i also prefer it that way